pkg/kor/kor.go

```diff
@@ -80,6 +82,78 @@ func GetKubeClient(kubeconfig string) *kubernetes.Clientset {
 	return clientset
 }
 
+func GetAPIExtensionsClient(kubeconfig string) *apiextensionsclientset.Clientset {
```

Comment:
This is very similar code to `GetKubeClient` function
Let's create a function that gets the config and we can create different clientsets based off this function

Same for dynamic client